content/dogepedia/articles/join-the-dogecoin-community.md

```diff
@@ -5,4 +5,8 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Talk about reddit, social networks, and how users can participate.
\ No newline at end of file
+There are many ways you can get involved in the Dogecoin community! Most popular social-media platforms have a thriving and welcoming Dogecoin community through which newcomers and old-timers alike can connect, discuss, and share their thoughts, memes, ideas, and spread the joy of all things Doge. 
+
+Reddit and Twitter are the most popular platforms for the Dogecoin community, with millions of active Shibes on either platform. On Reddit, for example, r/dogecoin is so popular that in 2021 it was ranked as Reddit’s Top community for cryptocurrency and finance, and posts in the Dogecoin subreddit are frequently the most Upvoted posts for the entire Reddit platform! If you’re more technically inclined, you can also check-out the technical development subreddit at r/dogecoindev. 

```

Comment:
```suggestion
**Reddit** and **Twitter** are the most popular platforms for the Dogecoin community, with millions of active Shibes on either platform. 

On Reddit, **[r/dogecoin](https://reddit.com/r/dogecoin/)** is so popular that in 2021 it was ranked as Reddit’s Top community for cryptocurrency and finance, and posts in the Dogecoin subreddit are frequently the most upvoted posts for the entire Reddit platform! If you are more technically inclined and want to stay updated about Dogecoin development, you can also check out the technical development subreddit at **[r/dogecoindev](https://reddit.com/r/dogecoindev/)**. **[r/dogeducation](https://reddit.com/r/dogeducation/)** is another subreddit where new Dogecoin users can find How-Tos and help.
```

Reply:
We could add bold for readability, links, and mentioned r/dogeducation (which could be a support to dogepedia, I think!)